src/agent/src/image_rpc.rs

```diff
@@ -26,8 +26,12 @@ use image_rs::image::ImageClient;
 use std::io::Write;
 
 const AA_PATH: &str = "/usr/local/bin/attestation-agent";
-const AA_KEYPROVIDER_PORT: &str = "127.0.0.1:50000";
-const AA_GETRESOURCE_PORT: &str = "127.0.0.1:50001";
+
+const AA_KEYPROVIDER_PORT: &str =
+    "unix:///opt/confidential-containers/attestation-agent/keyprovider.sock";
+const AA_GETRESOURCE_PORT: &str =
```

Comment:
Shouldn't those be renamed to `AA_KEYPROVIDER_URI` and `AA_GETRESOURCE_URI`?

Reply:
Oh, yes. Sorry for this. I'll fix this